core/swift41Action/swift4runner.py

```diff
@@ -66,7 +66,7 @@ def epilogue(self, init_message):
             with codecs.open(SRC_EPILOGUE_FILE, 'r', 'utf-8') as ep:
                 fp.write(ep.read())
 
-            fp.write('_run_main(mainFunction: %s)\n' % main_function)
+            fp.write('_run_main(mainFunction: %s)\n_ = _whisk_semaphore.wait(timeout: .distantFuture)\n' % main_function)
```

Comment:
Very minor nit;  I had to read this line a couple of times to understand what it was doing.  the "\n_=" runs together.  It might be clearer to do two fp.writes.

``` python
fp.write('_run_main(mainFunction: %s)\n' % main_function)
fp.write('_ = _whisk_semaphore.wait(timeout: .distantFuture)\n')
```

Reply:
Thanks @dgrove-oss I will update to do it in two lines for readability.